<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseTeamDialog</class>
 <widget class="QDialog" name="ChooseTeamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>321</width>
     <height>323</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>拜仁慕尼黑</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="radioButton_13">
         <property name="text">
          <string>瓦伦西亚</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>切尔西</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="radioButton_17">
         <property name="text">
          <string>AC米兰</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>尤文图斯</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="radioButton_19">
         <property name="text">
          <string>弗赖堡</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>阿森纳</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="radioButton_20">
         <property name="text">
          <string>巴黎圣日耳曼</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="radioButton_5">
         <property name="text">
          <string>利物浦</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QRadioButton" name="radioButton_14">
         <property name="text">
          <string>皇家贝蒂斯</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QRadioButton" name="radioButton_6">
         <property name="text">
          <string>曼联</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QRadioButton" name="radioButton_18">
         <property name="text">
          <string>多特蒙德</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QRadioButton" name="radioButton_7">
         <property name="text">
          <string>曼城</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QRadioButton" name="radioButton_15">
         <property name="text">
          <string>亚特兰大</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QRadioButton" name="radioButton_8">
         <property name="text">
          <string>西汉姆联</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" rowspan="2">
        <widget class="QRadioButton" name="radioButton_12">
         <property name="text">
          <string>马德里竞技</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" rowspan="2">
        <widget class="QRadioButton" name="radioButton_9">
         <property name="text">
          <string>埃弗顿</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" rowspan="2">
        <widget class="QRadioButton" name="radioButton_16">
         <property name="text">
          <string>国际米兰</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" rowspan="2">
        <widget class="QRadioButton" name="radioButton_10">
         <property name="text">
          <string>皇家马德里</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1" rowspan="2">
        <widget class="QRadioButton" name="radioButton_21">
         <property name="text">
          <string>勒沃库森</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QRadioButton" name="radioButton_11">
         <property name="text">
          <string>巴塞罗那</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChooseTeamDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChooseTeamDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
